server/requirements.in

```diff
@@ -3,7 +3,8 @@ honcho
 python3-saml>=1.9,<1.17
 python-xmp-toolkit>=2.0.1,<2.1
 num2words==0.5.13
+pyexiv2>=2.12.0,<2.13
```

Comment:
We can avoid this (and allow superdesk-core to define supported versions) using the `extras_required` from your [PR](https://github.com/superdesk/superdesk-core/pull/2598).

By using the following:
`Superdesk-Core[exiv2] @  git+https://github.com/superdesk/superdesk-core.git@develop#egg=superdesk-core` - [pip reference (example 7)](https://pip.pypa.io/en/latest/cli/pip_install/)